includes/variables.php

```diff
@@ -588,7 +588,8 @@
 //string url for the meeting finder, or false for no automatic archive page
 if (!isset($tsml_slug)) $tsml_slug = null;
 
```

Comment:
Here I'm just naming this anonymous function so it can be called below. The core issue was $tsml_slug wasn't set during first call to `tsml_custom_post_types()`